GUIDELINES.md

```diff
@@ -72,13 +72,12 @@ recipes. Recipes for dependencies with an `r-` prefix should be created using
 
 Add a wrapper script if the software is typically called via `java -jar ...`.
 For example, [fastqc](recipes/fastqc) already has a wrapper script, but
-[gatk-framework](recipes/gatk-framework) does not. [chromhmm](recipes/chromhmm)
-mimics the installation pattern of gatk-framework.
+[peptide-shaker](recipes/peptide-shaker) does not.
```

Comment:
Peptide Shaker has a wrapper script: https://github.com/bioconda/bioconda-recipes/tree/master/recipes/peptide-shaker
The new python one.